indexer/ftypes_matcher.cpp

```diff
@@ -498,7 +498,8 @@ AttractionsChecker::AttractionsChecker() : BaseChecker(2 /* level */)
       {"leisure", "park"},
       {"leisure", "water_park"},
       {"man_made", "lighthouse"},
-      {"man_made", "tower"},
+      {"man_made", "windmill"},
+      //{"man_made", "tower"},
```

Comment:
its also not clear what's the effect of the AttractionsChecker - could you add a comment please?

Reply:
Turned out - no mean full usage for now.